script/Ciel/Day 8/08_01_CIEL08_5.txt

```diff
@@ -582,7 +588,7 @@ The lone survivor takes this opportunity to address a question to her defenseles
 [sha:02c06a17abc3774fdf39863093d050c8ee0208c1]{
 -- Page 459, Offset 19415.
 -- なのにどうして、こんな……助けてやるなんて希望を持たせて殺しやがった!?
-So why... why are you killing us after you gave us false hope we'd be saved?!
+So then why... why would you give us false hope only to kill us?
```

Comment:
I know this was my suggestion and all, but in a way, false hope doesn't quite compute in this construction.
False hope implies it isn't real, so like, giving someone false hope and then killing them is very natural. But he's asking _why_, and if it's false hope, it's obvious why. You wouldn't need to ask; you already implicitly understand.

Though thinking about it this was a problem in the original construction too. Anyway...

>So then why... why would you give us the hope of salvation only to kill us?